### ISSUE TITLE
Script interface is broken. Need refractor for interface

### ISSUE THREAD

**Issue by hare1039** [hare1039/iwara-dl#8]
I want to make sure the interfaces are similar to other dl tools. Like `youtube-dl url => output one file` `megadl 'path' => write a file`, not in a `./dl/` directory. It actually breaks my daily updater after I fetch the new master branch...

So recently, I think we will refractor the `getopts` part to make the user interface more intuitive. (e.g `--opts value` etc)

If you have any suggestions, please write below. @momoandbanana22 

Currently, I think in the next version, we can
```
iwaradl.sh 
  -h --help       => show this help message and exit
  --username [U]  => username
  --userpass [P]  => password
  --load-ignore-list [File] => load the list of files that should not download
  -r --resume     => try resume download
  --retry [count] => Max time to retry the download fail
  --user          => treat input url as usernames
  --quiet-mode    => quiet mode
  --name-only     => output downloaded file name only(hides curl download bar)

  --shallow-update => only download users first page
  --updater-v1     => cd to each username folder; update each folder;
  --updater-v2     => create ./dl/ folder and update;

  --rm [File]      =>  add a name to iwara ignore list and delete the file

  --follow [M]  => Download videos of people you are following. M:MaxPage
  --dl-list [f] => Download using the VideoID in the [F] VideoID List file.
```

**Comment by momoandbanana22** [comment]
Your idea is great. I don't see any problem with it.
We need a new option switch.

**Comment by afterdelight** [comment]
add an option to only download new videos since last run

**Comment by Rika334** [comment]
Hello excuse me i wanna ask how befriending with someone pending on iwara . So please help me i confused

**Comment by hare1039** [comment]
@Rika334 I opened a new issue tracker => https://github.com/hare1039/iwara-dl/issues/13
We can discuss it there
